src/elements/collapse.svelte

```diff
@@ -29,45 +28,47 @@ const handleClick = (event: Event) => {
   isOpen = !isOpen;
   dispatch('toggle', { isOpen });
 };
-
 </script>
 
-<div class='relative w-full'>
+<div class="relative w-full">
```

Comment:
Suggestion: Since we're interspersing JS + HTML here could we have prettier using single quotes for everything? I'd rather not have to mentally switch between the two in the same file.

Reply:
Definitely agree, and attempted to do so by setting `jsxSingleQuote` to `true`, but ran right into https://github.com/sveltejs/prettier-plugin-svelte/issues/253.

Options, depending on how much we care about this:

1. Live with it until issue is resolved
2. Don't format `.svelte` files until issue is resolved
3. Ditch the PR 😭 

In my personal experience working in projects that used `'` in JS and `"` in JSX, I stopped thinking about it quite quickly once "format on save" was turned on, but YMMV

Reply:
I'm cool with 1., my only rec would be to link https://github.com/sveltejs/prettier-plugin-svelte/pull/271 somewhere in the eslint config so that we don't forget to track